src/main/java/step1/Operator.java

```diff
@@ -0,0 +1,41 @@
+package step1;
+
+import java.util.Arrays;
+
+public enum Operator {
+    PLUS("+") {
+        public int apply(int a, int b) {
+            return a + b;
+        }
+    },
+    MINUS("-") {
+        public int apply(int a, int b) {
+            return a - b;
+        }
+    },
+    MULTIPLY("*") {
+        public int apply(int a, int b) {
+            return a * b;
+        }
+    },
+    DIVIDE("/") {
+        public int apply(int a, int b) {
+            return a / b;
+        }
+    };
+
+    private final String symbol;
```

Comment:
```suggestion
public enum Operator {
    PLUS("+", (a, b) -> a + b),
    MINUS("-", (a, b) -> a - b),
    MULTIPLY("*", (a, b) -> a * b),
    DIVIDE("/", (a, b) -> a / b);

    private final String symbol;
    private final BiFunction<Integer, Integer, Integer> operation;
```
- 람다 표현식을 활용해서 보일러플레이트를 없애보는 건 어떨까요